src/components/ToastWrapper/ToastWrapper.tsx

```diff
@@ -1,8 +1,8 @@
-import './ToastWrapper.scss';
 import { ReactElement } from 'react';
+import './ToastWrapper.scss';
```

Comment:
could this scss styling be 'tailwindified' 😄 ?

Reply:
@mbaltrimas will create a separate ticket for this